artifacts/testdata/server/testcases/yara_detection.out.yaml

```diff
@@ -0,0 +1,51 @@
+LET _ <= SELECT
+      mock(plugin='info', results=[dict(OS='windows')]),
+      mock(plugin='http_client', results=[
+         dict(Url='http://remote',
+              Content=RemoteYara, Response=200)]),
+      -- First call to Windows.NTFS.MFT reveal target dll info
+      -- srcDir needs to be used to align fields
+      mock(artifact=Artifact.Windows.NTFS.MFT, results=[
+         dict(
+          EntryNumber=42344,
+          InUse = true,
+          ParentEntryNumber = 3589,
+          MFTPath = srcDir + "$MFT",
+          FullPath = "artifacts/testdata/files/wkscli.dll",
+          FileName = "wkscli.dll",
+          FileSize = 764976,
+          ReferenceCount = 2,
+          IsDir = false,
+          Created0x10 = "2020-11-19T02:48:46.9885373Z",
+          Created0x30 = "2021-01-24T06:57:36.7073284Z",
+          LastModified0x10 = "2020-11-19T02:48:47.0041758Z",
+          LastModified0x30 = "2021-01-24T06:57:39.5353986Z",
+          LastRecordChange0x10 = "2021-01-24T06:59:40.7021579Z",
+          LastRecordChange0x30 = "2021-01-24T06:57:39.5353986Z",
+          LastAccess0x10 = "2021-03-05T13:44:04.1999087Z",
+          LastAccess0x30 = "2021-01-24T06:57:36.7073284Z")
+      ])
+FROM scope()
+[]SELECT FullPath,Size,Rule,Rule,Meta,HitContext,HitOffset FROM Artifact.Windows.Detection.Yara.NTFS(PathRegex='wkscli.dll',ShortHandYara='wide ascii:MZ')[
+ {
+  "FullPath": "d:\\a\\velociraptor\\velociraptor\\artifacts/testdata/files/wkscli.dll",
```

Comment:
We need to remove the leading directory here or it wont be able to run locally. Try to pass here 

`rel(path=FullPath, base=srdDir)`

Reply:
Local works ok as my mocked MFT result is: MFTPath = srcDir + "$MFT".
The FullPath is calculated with: split(string=MFTPath,sep='\\$')[0] + FullPath as FullPath